test/extended/testdata/jenkins-plugin/build-with-exec-steps.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project>
+    <actions/>
+    <description></description>
+    <keepDependencies>false</keepDependencies>
+    <properties>
+        <io.fabric8.jenkins.openshiftsync.BuildConfigProjectProperty plugin="openshift-sync@0.0.13">
+            <uid></uid>
+            <namespace></namespace>
+            <name></name>
+            <resourceVersion></resourceVersion>
+            <contextDir></contextDir>
+        </io.fabric8.jenkins.openshiftsync.BuildConfigProjectProperty>
+    </properties>
+    <scm class="hudson.scm.NullSCM"/>
+    <canRoam>true</canRoam>
+    <disabled>false</disabled>
+    <blockBuildWhenDownstreamBuilding>false</blockBuildWhenDownstreamBuilding>
+    <blockBuildWhenUpstreamBuilding>false</blockBuildWhenUpstreamBuilding>
+    <triggers/>
+    <concurrentBuild>false</concurrentBuild>
+    <builders>
+        <com.openshift.jenkins.plugins.pipeline.OpenShiftExec plugin="openshift-pipeline@1.0.31-SNAPSHOT">
+            <apiURL></apiURL>
+            <namespace>PROJECT_NAME</namespace>
```

Comment:
I believe the plugin will default to the namespace running Jenkins. In the present test harness, Jenkins runs in a namespace separate from the namespace devoted to each test spec.
This effectively forces cross namespace operations and has managed to shake out a couple of defects.